github/client.go

```diff
@@ -371,6 +371,20 @@ func (client *Client) DeleteReleaseAsset(asset *ReleaseAsset) (err error) {
 	return
 }
 
+func (client *Client) DownloadReleaseAsset(url string) (asset io.ReadCloser, err error) {
+	api, err := client.simpleApi()
+	if err != nil {
+		return
+	}
+
+	asset, err = api.GetBody(url, "application/octet-stream")
```

Comment:
@mislav `Get()` doesn't appear to allow setting the Accept header. Should I change it's signature?